design-reference/docs/azure-simple.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Azure Simple
+draft: true
+date: 2019-05-18T13:09:18.252Z
+author:
+  name: Rachel Weil
+  github-username: hxlnt
+sourceUrl: https://github.com/microsoft/cobalt/
+tags:
+  - app-insights
+  - app-gateway
+  - azure
+  - monitoring
+  - traffic-manager
+targetUser:
+  - app-dev
+category: template
+providers:
+  - azure
+---
+
+# Azure Simple
+
+Summary of template and its use cases. Required.
+
+<!---
+This content was generated automatically using filename of this document.
+--->
+
+![Architecture diagram](azure-simple-architecture.png)
+
+<!---
+This is a standard Table of Contents. Remove any optional sections that do not contain content.)
+--->
+
+**Table of Contents**
+ - [Quickstart](#Quickstart)
```

Comment:
What are your thoughts with the below ordering?

1. Design rationale
2. Business Use-Case
3. Module Listing
4. Deployment Steps
5. Features
    - Performance Load Testing
    - Scaling
    - Cost
    - Security
    - Authentication
    - Authorization
    - DevOPS Pipelines
    - Monitoring
6. Local Setup + Testing
7. Module configuration

Reply:
I am a strong advocate for having the first section be a quickstart with the bare minimum step-by-step instructions all in one place. Having to scroll around to different sections to find the instructions is frustrating for the user.

Reply:
Per conversation with @erikschlegel, I've split out "Quickstart" into its own Markdown file. (In reality, it will be an .html template, but it's in .md now for ease of review.)